internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerRoute.java

```diff
@@ -120,45 +124,83 @@ public enum ControllerRoute {
   ), SET_VERSION("/set_version", HttpMethod.POST, Arrays.asList(NAME, VERSION)),
   ROLLBACK_TO_BACKUP_VERSION(
-      "/rollback_to_backup_version", HttpMethod.POST, Collections.singletonList(NAME), REGIONS_FILTER
-  ), AGGREGATED_HEALTH_STATUS("/aggregatedHealthStatus", HttpMethod.POST, Collections.emptyList()),
+      "/rollback_to_backup_version", HttpMethod.POST, Collections.singletonList(NAME),
+      Collections.singletonList(REGIONS_FILTER)
+  ),
+  AGGREGATED_HEALTH_STATUS(
+      "/aggregatedHealthStatus", HttpMethod.POST, Collections.emptyList(), Collections.emptyList(), false
+  ),
   ROLL_FORWARD_TO_FUTURE_VERSION(
-      "/roll_forward_to_future_version", HttpMethod.POST, Collections.singletonList(NAME), REGIONS_FILTER
+      "/roll_forward_to_future_version", HttpMethod.POST, Collections.singletonList(NAME),
+      Collections.singletonList(REGIONS_FILTER)
   ),
   // Enable/disable read write for this store. Status is "true" or "false". Operation "read" or "write" or "readwrite".
   ENABLE_STORE("/enable_store", HttpMethod.POST, Arrays.asList(NAME, OPERATION, STATUS)),
   DELETE_ALL_VERSIONS("/delete_all_versions", HttpMethod.POST, Collections.singletonList(NAME)),
   DELETE_OLD_VERSION("/delete_old_version", HttpMethod.POST, Arrays.asList(NAME, VERSION)),
   UPDATE_CLUSTER_CONFIG(
       "/update_cluster_config", HttpMethod.POST, Collections.singletonList(CLUSTER),
-      SERVER_KAFKA_FETCH_QUOTA_RECORDS_PER_SECOND
+      Collections.singletonList(SERVER_KAFKA_FETCH_QUOTA_RECORDS_PER_SECOND)
   ),
 
   JOB("/job", HttpMethod.GET, Arrays.asList(NAME, VERSION)),
   KILL_OFFLINE_PUSH_JOB("/kill_offline_push_job", HttpMethod.POST, Collections.singletonList(TOPIC)),
-  LIST_STORES("/list_stores", HttpMethod.GET, Collections.emptyList(), INCLUDE_SYSTEM_STORES),
-  LIST_CHILD_CLUSTERS("/list_child_clusters", HttpMethod.GET, Collections.emptyList()),
+  LIST_STORES(
+      "/list_stores", HttpMethod.GET, Collections.emptyList(), Collections.singletonList(INCLUDE_SYSTEM_STORES)
+  ), LIST_CHILD_CLUSTERS("/list_child_clusters", HttpMethod.GET, Collections.emptyList()),
   LIST_NODES("/list_instances", HttpMethod.GET, Collections.emptyList()),
   CLUSTER_HEALTH_STORES("/cluster_health_stores", HttpMethod.GET, Collections.emptyList()),
   ClUSTER_HEALTH_INSTANCES(
-      "/cluster_health_instances", HttpMethod.GET, Collections.emptyList(), ENABLE_DISABLED_REPLICAS
+      "/cluster_health_instances", HttpMethod.GET, Collections.emptyList(),
+      Collections.singletonList(ENABLE_DISABLED_REPLICAS)
   ), LIST_REPLICAS("/list_replicas", HttpMethod.GET, Arrays.asList(NAME, VERSION)),
   NODE_REPLICAS("/storage_node_replicas", HttpMethod.GET, Collections.singletonList(STORAGE_NODE_ID)),
   NODE_REMOVABLE(
-      "/node_removable", HttpMethod.GET, Collections.singletonList(STORAGE_NODE_ID), INSTANCE_VIEW,
-      TO_BE_STOPPED_INSTANCES
+      "/node_removable", HttpMethod.GET, Collections.singletonList(STORAGE_NODE_ID),
```

Comment:
This can also not have sensitive body ?

Reply:
This endpoint doesn't have have any body, right?

Reply:
I realize that `bodyHasSensitiveContent` might not be the right variable to decide if body should be logged or not. I wanted to not make a very specific config for audit logging. But I'm open to suggestions if you have better ones.